src/Components/App/App.js

```diff
@@ -3,17 +3,25 @@ import './App.css';
 import JoinForm from '../JoinForm/JoinForm.js';
 import RoomForm from '../RoomForm/RoomForm.js';
 import { Route } from 'react-router-dom';
+import {ReactComponent as ChatLogo} from './assets/message.svg'
 
 function App() {
   return (
-    <div className="App">
+    <main className="App">
```

Comment:
div/vs main--why is this change necessary?

Reply:
Semantic for screen readers